index.js

```diff
@@ -105,7 +105,12 @@ class Ky {
 			this._options.prefixUrl += '/';
 		}
 
-		this._input = this._options.prefixUrl + this._input;
+		const url = new URL(this._options.prefixUrl + this._input);
+		if (searchParams) {
+			url.search = searchParams instanceof URLSearchParams ? searchParams : new URLSearchParams(searchParams).toString();
```

Comment:
What about:

```js
if (typeof searchParams === 'string') {
    url.search = searchParams;
} else {
    url.search = new URLSearchParams(searchParams).toString();
}
```

Reply:
Your suggestion looks good to me, except for the fact that if `searchParams` is an instance of `URLSearchParams` you end up  initiating a clone of it for no reason.
We could do this:
```js
if (typeof searchParams === 'string' || searchParams instanceof URLSearchParams) {
    url.search = searchParams;
} else {
    url.search = new URLSearchParams(searchParams).toString();
}
```
Which both doesn't parse strings nor `URLSearchParams` objects.

Reply:
Lemme fix your code:

```js
if (typeof searchParams === 'string') {
    url.search = searchParams;
} else if (searchParams instanceof URLSearchParams) {
    url.search = searchParams.toString();
} else {
    url.search = new URLSearchParams(searchParams).toString();
}
```

I hope it works as expected

Reply:
We could do that, but from my testing in both Node and my browser (Chrome), ` url.search = searchParams` works.

This is how I tested it:

![screenshot](https://user-images.githubusercontent.com/22768990/47598334-0913f680-d9a3-11e8-962b-1bca6c4b0add.png)

Reply:
In a flash ;)

Reply:
I ended up doing:
```js
if (typeof searchParams === 'string' || searchParams instanceof URLSearchParams) {
	url.search = searchParams;
} else if (searchParams) {
	url.search = new URLSearchParams(searchParams).toString();
}
```

Since if the `searchParams` option isn't defined, the test `prefix-url › prefixUrl option` fails (which makes sense).